test/jdk/java/awt/Frame/FrameStateTest/FrameStateTest.java

```diff
@@ -119,7 +122,7 @@ public static void main(String[] args) throws Exception {
                 .title("GetBoundsResizeTest Instructions")
```

Comment:
Looks like title is mismatched with the test. Should it be "Frame State and Size Test Instructions" ?

Reply:
Updated.